clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -573,6 +573,11 @@ struct EmptyCoverageMappingBuilder : public CoverageMappingBuilder {
 /// creation.
 struct MCDCCoverageBuilder {
 
+  struct DecisionIDPair {
+    MCDCConditionID TrueID = 0;
+    MCDCConditionID FalseID = 0;
+  };
```

Comment:
Initializers are no longer required since all constructions are made explicitly.

Reply:
I've forgot initial values are used as the default value in `createBranchRegion`. Sorry.